examples/hotrod/Dockerfile

```diff
@@ -1,9 +1,13 @@
 # Copyright (c) 2024 The Jaeger Authors.
 # SPDX-License-Identifier: Apache-2.0
 
+FROM alpine:latest AS prep
+RUN apk --update add ca-certificates
```

Comment:
see `docker/base/Dockerfile` for how we normally do it

Reply:
updated to the same